src/app/inventory/ConnectedInventoryItem.tsx

```diff
@@ -32,18 +33,22 @@ interface StoreProps {
 }
 
 function mapStateToProps(state: RootState, props: ProvidedProps): StoreProps {
-  const { item } = props;
-
+  const { item, searchFilterIsEmpty } = props;
   const settings = settingsSelector(state);
   const itemInfos = itemInfosSelector(state);
   const itemHashTags = itemHashTagsSelector(state);
+  const tag = getTag(item, itemInfos, itemHashTags);
 
   return {
     isNew: settings.showNewItems ? state.inventory.newItems.has(item.id) : false,
-    tag: getTag(item, itemInfos, itemHashTags),
+    tag,
     notes: getNotes(item, itemInfos, itemHashTags) ? true : false,
     wishlistRoll: wishListSelector(item)(state),
-    searchHidden: props.allowFilter && !searchFilterSelector(state)(item),
+    searchHidden:
+      // dim this item if there's no search filter and it's archived
+      (searchFilterIsEmpty && tag === 'archive') ||
+      // or if there is filtering and it doesn't meet the condition
+      (props.allowFilter && !searchFilterSelector(state)(item)),
```

Comment:
You could extract `searchFilterSelector(state)` into a var and not have to pass in `searchFilterIsEmpty`, then I guess the prop just becomes whether or not to apply the archive dimming.

Reply:
that's a lot cleaner, no idea what i was thinking